funnel/views/schedule.py

```diff
@@ -145,7 +145,7 @@ class ProjectScheduleView(ProjectViewMixin, UrlForView, ModelView):
     def schedule(self):
         schedule_transition_form = ProjectScheduleTransitionForm(obj=self.obj)
         return dict(project=self.obj,
-            from_date=date_js(self.obj.date), to_date=date_js(self.obj.date_upto),
+            from_date=date_js(self.obj.schedule_start_at), to_date=date_js(self.obj.schedule_end_at),
```

Comment:
The data type is changing from date to datetime. Does this affect the template?

Reply:
It doesn't. Previous they were timestamps created from `date` object, now they are timestamps created from `datetime` object.

Reply:
Looks like they need to be cast into the project's timezone though.